### ISSUE TITLE
Dry-run during delete snapshots shows errant or broken output

### ISSUE THREAD

**Issue by dbeckham** [elastic/curator#293]
Running a `delete snapshots` command against a repo outputs what appears to be broken log info.

This command:
```
curator --dry-run --host es.example.com --port 9200 delete snapshots --repository myrepo --time-unit days --older-than 30
```

Resulted in the following output:
```
2015-03-10 10:21:45,926 INFO      Job starting...
2015-03-10 10:21:45,926 INFO      DRY RUN MODE.  No changes will be made.
2015-03-10 10:21:47,833 WARNING   DRY RUN: Will not perform delete action
curator-20150208230022
```
The snapshot `curator-20150208230022` looks to be within the time range for deletion, but the output appears broken or incomplete.

**Comment by dbeckham** [comment]
The output is definitely broken. If I run the same command with 20 days, I see the following output:

```
2015-03-10 10:26:06,208 INFO      Job starting...
2015-03-10 10:26:06,208 INFO      DRY RUN MODE.  No changes will be made.
2015-03-10 10:26:08,866 WARNING   DRY RUN: Will not perform delete action
curator-20150208230022
curator-20150209230022
curator-20150210230029
curator-20150211230025
curator-20150212230025
curator-20150213230026
curator-20150214230026
curator-20150215230030
curator-20150216230024
curator-20150217230024
curator-20150218230025
```

**Comment by untergeek** [comment]
What does the output reveal if you issue the `show` command (without the `--dry-run` flag) instead of delete?  It seems to me that the output is correct in both examples.  It's showing which snapshots would be deleted, and announcing that it will not actually delete them.

**Comment by untergeek** [comment]
If you were looking for log output similar to Curator v2, I apologize.  It is a major point release and much has been changed, including this.

Because of the new filtered pipeline approach to index and snapshot selection, I do not do the logging the same as in the previous version.

**Comment by dbeckham** [comment]
I think it's two things.  I actually ended up running the show later and saw that the last line with a datestamp indicated that what follows was a list of the snapshots found. The `delete snapshots` command output above doesn't use similar language.  Because of the sudden change in the log format being output for the command and that it didn't tell me it found these snapshots, it looked like debug or unintentional output. 

I realized after running show and running this command with different parameters, that you intended to not have a datestamp or log level for that output, so it's mostly just that the dry run output should be a bit clearer like the show command is.

Honestly, this is a pretty minor problem. I was trying to work through the command line differences between 2.0 and 3.0 and I was confused by some of the output where I was expecting something different than what I got.
